client/assets/js/scoreboard.js

```diff
@@ -1,19 +1,45 @@
 import $ from 'jquery';
 
-export default function scoreBoard(snakes) {
-	snakes.forEach(function(e){
-		var li = $("<li>");
-		li.css( "color", e.color);
+export default class Scoreboard {
+
+	addPlayer(snake) {
+		$("#userlist").append(this.createPlayer(snake));
```

Comment:
Or rather, if ScoreBoard is to become a class, it should create its container and not use a unique one already in the DOM tree